PolyEngine/Engine/Src/ResourceManager.cpp

```diff
@@ -14,26 +14,29 @@ DEFINE_RESOURCE(TextureResource, gTextureResourcesMap)
 DEFINE_RESOURCE(FontResource, gFontResourcesMap)
 DEFINE_RESOURCE(SoundResource, gALSoundResourcesMap)
 
-ENGINE_DLLEXPORT String Poly::LoadTextFileRelative(eResourceSource Source, const String & path)
+String Poly::LoadTextFileRelative(eResourceSource Source, const String & path)
 {
 	bool IsNotLoaded = true;
 
 	String FileContent;
-	Dynarray<String> Paths = (Source == eResourceSource::NONE) ? Dynarray<String>{""} : gAssetsPathConfig.GetAssetsPaths(Source);
-	for (size_t i = 0; i < Paths.GetSize() && IsNotLoaded; ++i)
+	String AbsolutePath = gAssetsPathConfig.GetAssetsPath(Source) + path;
```

Comment:
This depends on paths having a trailing separator, which might not always be the case

Reply:
`Path` wrapper is planned. It will fix this kind of problems.

Reply:
Right. But we do not have it yet and so the code is currently subtly broken.